docs/tbdev_getting_started.ipynb

```diff
@@ -221,11 +220,15 @@
         "id": "NyJsD3Ypyx6m"
       },
       "source": [
-        "### Screenshot of TensorBoard.dev\n",
+        "### Screenshots of TensorBoard.dev\n",
         "\n",
-        "This is what it will look like when you navigate to your new experiment on TensorBoard.dev.\n",
+        "This is what it will look like when you navigate to TensorBoard.dev experiment list:\n",
```

Comment:
Suggestion. Modify:

This is what it will look like when you navigate to TensorBoard.dev experiment list:

To:

This is what it will look like when you navigate to https://tensorboard.dev/experiments/

Reply:
Modified. Thanks!